src/global/pidfile.cc

```diff
@@ -158,9 +164,6 @@ int pidfile_open(const md_config_t *conf)
     pfh.close();  
```

Comment:
@dachary shall we `::open(pfh.pf_path, O_CREAT|O_WRONLY|O_TRUNC, 0644);` instead?

otherwise when we are rewriting the pid, not all the existing bits are overwritten. and the resulting pid might be misleading.